### PR TITLE
setup file added

### PR DESCRIPTION
**Pull request by Andrsi13**

### CODE REVIEW

**Comment by svilav** [comment]
Try renaming `console_bot.py` to anything else not be the same as the directory

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      version='1.1',
+      description='Console bot that will help you with everything',
+      url='https://github.com/TetianaFilonenko/project-team-4.git',
+      author='Kobzar Coders',
+      author_email='flyingcircus@example.com',
+      license='MIT',
+      packages= find_namespace_packages(),
+      install_requires= ['faker', 'prompt_toolkit'],
+      include_package_data=True,
+      entry_points= { 'console_scripts': ["run = console_bot:main"]}
```
Comment (svilav) [inline setup.py]: `run` is the command used in the terminal to call our app, right? If yes, plz rename it to something meaningful, e.g. `run-console-bot`

**Comment by Andrsi13** [comment]
замінив console_bot.py на main.py
Помилки лишились
image

**Comment by svilav** [comment]
Based on what your error message tells, I created new file called `__main__` with the following code:
```
from console_bot import main

if __name__ == "__main__":
    main()
```
Seems to work

**Pull request closed without merging**